synapse/tests/test_lib_stormsvc.py

```diff
@@ -322,9 +320,9 @@ class StormSvcTest(s_test.SynTest):
                 await core.nodes(f'service.add fake {lurl}')
                 await core.nodes('$lib.service.wait(fake)')
 
-                core.svcsbyname['fake'].client._t_conf['timeout'] = 0.1
+                core.svcsbyname['fake'].proxy._t_conf['timeout'] = 0.1
```

Comment:
nitpicking - it feels wrong to change the name of the client objects to proxy when they are distinct things.

Reply:
This is due to the inheritance of the svc object from stormtypes.Proxy